sourcecode/apis/common/app/Providers/AppServiceProvider.php

```diff
@@ -19,11 +21,18 @@ public function register()
             AuthApiService::class,
             fn($app) => new AuthApiService(),
         );
-
         $this->app->bind(
             ResourceApiService::class,
             fn($app) => new ResourceApiService(),
         );
+        $this->app->bind(
```

Comment:
It's not necessary to explicitly register services without ambiguous dependencies. Laravel does this automatically.